prow/plugins/trigger/ic.go

```diff
@@ -65,12 +65,16 @@ func handleIC(c client, ic github.IssueCommentEvent) error {
 		return err
 	}
 
+	trustedOrg := org
```

Comment:
This code looks like it fails "open-ish".  If the config isn't set or the repo in question isn't in the map, then it defaults to trusting that repo with itself.  This is a change in behavior from before.  Previously, the k8s org was used regardless.

I'd have another config parameter called "default trusted org" and use that.  Set that to `kubernetes`.  If it isn't set, then consider that there is no default and for non-listed repos there are no users that are trusted.